### ISSUE TITLE
When using sub-structs fluent creates only the first table is sub-names are same

### ISSUE THREAD

**Issue by rafiki270** [vapor/fluent#350]
```swift
extension Token.Full {
    
    public static var name: String = "Token.Full"
    public static var entity: String = "tokens"
    
    public typealias Database = MySQLDatabase
    public typealias ID = Int
    
    public static var idKey = \Token.Full.id
    
}

extension User.Full {
    
    public static var name: String = "User.Full"
    public static var entity: String = "users"
    
    public typealias Database = MySQLDatabase
    public typealias ID = Int
    
    public static var idKey = \User.Full.id
    
}

// Only Token will be created as `Full` already exists in the fluent table
migrationConfig.add(model: Token.Full.self, database: .db)
migrationConfig.add(model: User.Full.self, database: .db)
```

**Comment by twof** [comment]
What do the two `Full` structs look like?

**Comment by tanner0101** [comment]
@rafiki270 can you paste the contents of the `fluent` table in your DB? I'm curious what is be stored there.

**Comment by tanner0101** [comment]
This will be fixed by #428 since model's use the `name` property now for migration names 👍
